src/languages/php.js

```diff
@@ -313,7 +317,7 @@ export default function(hljs) {
           regex.concat(WHITESPACE, "+"),
           // to prevent built ins from being confused as the class constructor call
           regex.concat("(?!", normalizeKeywords(BUILT_INS).join("\\b|"), "\\b)"),
-          /\\?\w+/,
+          regex.concat("\\\\?", IDENT_RE),
```

Comment:
Can you point me to the test case this is covering?  I didn't see anything with a `\?`

Reply:
[new \DateTime()](https://github.com/highlightjs/highlight.js/pull/3422/files#diff-60c00e73629094e4af71f4387ed9c01356d51223ba5cf0e8ad21a792426efe07R25)

Reply:
Ah I see  the markup test for it now.